src/nwb/ecephys/ElectricalSeries.hpp

```diff
@@ -109,17 +110,27 @@ class ElectricalSeries : public TimeSeries
                Base unit of measurement for working with the data. 
                This value is fixed to volts)
 
-  DEFINE_FIELD(readElectrodes,
-               DatasetField,
-               int,
-               "electrodes",
-               The electrodes that generated this electrical series.)
+  DEFINE_FIELD(
+      readElectrodes,
+      DatasetField,
+      int,
+      "electrodes",
+      The indices of the electrodes that generated this electrical series.)
 
   DEFINE_FIELD(readElectrodesDescription,
                AttributeField,
                std::string,
                "electrodes/description",
                The electrodes that generated this electrical series.)
+
+  DEFINE_REFERENCED_REGISTERED_FIELD(
+      readElectrodesTable,
+      ElectrodeTable,
+      "electrodes/table",
+      Table with all extracellular electrodes retrieved from the reference to
+              the electrode table stored in the `electrodes
+          / table` attribute)
```

Comment:
```suggestion
     The electrodes table retrieved from the object referenced in the 
      `electrodes/table` attribute.)
```

I found the description a little hard to understand, tried to clarify